src/Build/BackEnd/Components/SdkResolution/SdkResolverService.cs

```diff
@@ -191,11 +211,27 @@ private SdkResult ResolveSdkUsingResolversWithPatternsFirst(int submissionId, Sd
                 projectPath,
                 interactive,
                 isRunningInVisualStudio,
-                out sdkResult))
+                out sdkResult,
+                out IEnumerable<string> moreErrors,
+                out IEnumerable<string> moreWarnings))
             {
                 return sdkResult;
             }
 
+            errors.AddRange(moreErrors);
+            warnings.AddRange(moreWarnings);
+
+            if (throwExceptions)
+            {
+                loggingContext.LogError(new BuildEventFileInfo(sdkReferenceLocation), "FailedToResolveSDK", sdk.Name);
+            }
+
+            LogWarnings(loggingContext, sdkReferenceLocation, warnings);
```

Comment:
Would not it be confusing to have warnings after the message `FailedToResolveSDK` above and before the actual list of errors below? Maybe move this line above or below?

Reply:
That is a good point...I will talk with baronfel about it to see what he recommends.